src/test/java/teammates/test/cases/browsertests/InstructorHomePageUiTest.java

```diff
@@ -329,6 +329,9 @@ private void testPublishUnpublishActions() {
         homePage.verifyUnclickable(homePage.getPublishLink(feedbackSessionAwaiting.getCourseId(),
                                                            feedbackSessionAwaiting.getFeedbackSessionName()));
```

Comment:
I would suggest this line to be removed since the link would still be clickable when disabled. Testing whether it is disabled or not is redundant.